settings/Core.setting.php

```diff
@@ -1066,4 +1066,23 @@
     'description' => ts('Acceptable Mime Types that can be used as part of file urls'),
     'help_text' => NULL,
   ],
+  'http_timeout' => [
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'http_timeout',
+    'type' => 'Integer',
+    'quick_form_type' => 'Element',
+    'html_type' => 'text',
+    'html_attributes' => [
+      'size' => 2,
+      'maxlength' => 3,
+    ],
+    'default' => 5,
+    'add' => '5.15',
```

Comment:
@seamuslee001 should this be 5.14?

Reply:
I wasn't aware it was going to be backported so that is why i put 5.15 @eileenmcnaughton